CommandTests/Generated/swift_packages_verbose.2.0494a7b3.md

```diff
@@ -0,0 +1,64 @@
+# Command
+```json
+["-p1", "{ios_project_with_swift_packages_1}", "-p2", "{ios_project_with_swift_packages_2}", "-v"]
+```
+
+# Expected exit code
+2
+
+# Expected output
+```
+✅ FILE_REFERENCES
+✅ BUILD_PHASES > "App" target
+✅ COPY_FILES > "App" target
+✅ TARGETS > NATIVE targets
+✅ TARGETS > AGGREGATE targets
+✅ HEADERS > "App" target
+✅ SOURCES > "App" target
+✅ RESOURCES > "App" target
+✅ CONFIGURATIONS > Root project
+✅ SETTINGS > Root project > "Debug" configuration > Base configuration
+✅ SETTINGS > Root project > "Debug" configuration > Values
+✅ SETTINGS > Root project > "Release" configuration > Base configuration
+✅ SETTINGS > Root project > "Release" configuration > Values
+✅ SETTINGS > "App" target > "Debug" configuration > Base configuration
+✅ SETTINGS > "App" target > "Debug" configuration > Values
+✅ SETTINGS > "App" target > "Release" configuration > Base configuration
+✅ SETTINGS > "App" target > "Release" configuration > Values
+✅ SOURCE_TREES > Root project
+✅ DEPENDENCIES > "App" target
+❌ LINKED_DEPENDENCIES > "App" target
+
+⚠️  Only in second (1):
+
+  • Swinject
+
+
+⚠️  Value mismatch (2):
+
+  • ComposableArchitecture package reference
+    ◦ version = .upToNextMajorVersion(0.9.0)
+    ◦ version = .upToNextMinorVersion(0.9.0)
```

Comment:
There is potentially some redundancy here as the same information can be seen within `SWIFT_PACKAGES` section. The reason I opted for adding it is to cater for the scenario where a package product doesn't have a source package specified (e.g. like the Swifter example below) and that can only be highlighted here as it's a difference at the linked dependency level.